.github/workflows/cln-plugin.yaml

```diff
@@ -1,10 +1,14 @@
 name: CI tests for CLN watchtower-plugin
 
-on: [push, pull_request]
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
 
 env:
-  bitcoind_version: 0.20.1
-  cln_version: 0.12.1
+  bitcoind_version: "27.0"
+  cln_version: "24.02.2"
```

Comment:
Is `24.02.2` any different from `24.2.2` (as the packages used in watchtower-plugin/tests/pyproject.toml (ref next comment))?

for me `02` feels *smaller* for some reason. Let's stick with just `2` here.

Reply:
yes it is different: one is the CLN version the other the pyln-* version. These are not identical and there is no CLN 24.2.2